src/app/content/components/Page.tsx

```diff
@@ -168,9 +168,10 @@ export class PageComponent extends Component<PropTypes> {
 
 export const contentTextStyle = css`
   ${bodyCopyRegularStyle}
+@media screen { /* full page width in print */
   max-width: ${contentTextWidth}rem;
   margin: 0 auto;
```

Comment:
i think this margin is somehow overriding the margin in the PrevNextBar here https://github.com/openstax/rex-web/blob/master/src/app/content/components/PrevNextBar.tsx#L47 you can see that component is pulling in `contentTextStyle` but applying its own margin after.

if you compare this branch to master, you see that these two styles, the max-width and margin, are now in their own class instead of being applied directly to the same class when you drop in the variable. maybe something to do with the css pre-processor handling the `@media` call? we may want to re-think how we use variables to dry styles like this in the future.

for now, since this `contentTextStyle` is used by other things like attribution, prev/next links, and all you really care about is the page component, maybe you could put a `@media print` override down in the component on line 177?